src/on-tour/components/Totals/Totals.js

```diff
@@ -46,18 +46,24 @@ export default class Totals extends Component {
   }
 
   distanceTotal() {
-    // locationDistances(this.props.Locations)
-    let currentGigs = this.props.Locations
-    let totalMiles = parseInt(currentGigs.reduce((distanceAcc, gig) => {
-      return distanceAcc + gig.distance_from_last
+    let updatedLocations = locationDistances(this.props.Locations)
```

Comment:
If you're not reassigning updatedLocations and totalMiles, they could be const instead of let